xbmc/video/windows/GUIWindowVideoBase.cpp

```diff
@@ -763,7 +763,7 @@ std::string CGUIWindowVideoBase::GetResumeString(const CFileItem &item)
 
 bool CGUIWindowVideoBase::ShowResumeMenu(CFileItem &item)
 {
-  if (!item.m_bIsFolder && !item.IsPVR())
+  if (!item.m_bIsFolder && !item.IsPVR() || item.IsPlugin())
```

Comment:
Ok, I will remove this change.
So do you want me to try to find the root cause for this?